pkgs/by-name/ta/tauon/package.nix

```diff
@@ -54,10 +54,14 @@ python3Packages.buildPythonApplication rec {
       --replace-fail 'base_path = Path(pctl.install_directory).parent.parent / "build"' 'base_path = Path("${placeholder "out"}/${python3Packages.python.sitePackages}")'
   '';
 
+  dontUsePythonRuntimeDepsCheck = true;
```

Comment:
please add a comment about why this is here, so others know when it can be re-enabled